### ISSUE TITLE
Schamane Felsbeißer

### ISSUE THREAD

**Issue by Soox1991** [ClassicWoW/Nefarian_1.12.1_Bugtracker#1624]
**Welches Verhalten wird Beobachtet:**
[Felsbeisser](http://datenbank.classic-wow.org/?spell=10399) procct ab und an den Zauber [Flammenzunge.](http://datenbank.classic-wow.org/?spell=8024)

**Wie sollte es sich verhalten?**
Jeder Rang von Felsbeisser startet einen Waffenproc der dazu bestimmt ist einen festen Wert an Bedrohung pro Sekunde zu erzeugen. Im höchsten Rang gibt Felsbeisser 72 Bedrohung pro Sekunde bei erfolgreichen Treffern. Bei einer Waffe mit Geschwindigkeit 3.0 erzeugt jeder Treffer (jedoch kein verfehlen, ausweichen oder parieren) 3*72=216 Bedrohung.

**Schritte zur Reproduzierung:**
Felsbeisser buffen danach Infight gehen

Zusätzliche Informationen:
http://www.razyboard.com/system/morethread-das-abc-der-aggro-cryper-1284148-3416836-0.html
(17.07.2006)
http://wow-mow.de.tl/Shamanen-Guide.htm (unter 2.14) (08.09.2006)
http://datenbank.classic-wow.org/?spell=10399
http://datenbank.classic-wow.org/?spell=8024

**Comment by Badorr** [comment]
Siehe:
https://github.com/ClassicWoW/Nefarian_1.12.1_Bugtracker/issues/1060
https://github.com/ClassicWoW/Nefarian_1.12.1_Bugtracker/issues/1299

Das mit der 72 Aggro pro Sekunde würde ich mal genauer unter die Lupe nehmen. Wird nämlich definitiv nicht im KTM angezeigt, aber man hat definitiv mehr Aggro.

**Comment by Sabbaer** [comment]
tank schami incoming? müsste man mal probieren ;)

**Comment by hraesvelgr** [comment]
@Badorr ktm sollte man nicht zu 100% vertrauen. es zeigt beim paladin ebenfalls falsche werte an (zorn der gerechtigkeit wird mit mit 90% statt 60%, und120% statt 90% verrechnet (wenn geskillt) soweit ich mich erinner). die korrekten aggrowerte funktionieren aber auf unserem server. ist nur ein anzeigefehler/berechnungsfehler in ktm

**Comment by Haudraufinix** [comment]
Ich glaub der Bug tritt nur bei den Rängen 1-6 auf und löst den jeweiligen Rang Flammenzunge aus. Rang 7 Felsbeisser scheint diesen Fehler nicht auszulösen.

**Comment by Badorr** [comment]
Nein, tritt auch bei Rang 7 Felsbeißer auf. Steht ebenfalls im Kampflog drin. Dem Schaden des Flammenzungenproccs nach war es ebenfalls der höchste Rang (Rang 6)
